frontend/pages/g/_groupSlug/r/create/url.vue

```diff
@@ -4,7 +4,8 @@
       <div>
         <v-card-title class="headline"> {{ $t('recipe.scrape-recipe') }} </v-card-title>
         <v-card-text>
-          {{ $t('recipe.scrape-recipe-description') }}
+          <p>{{ $t('recipe.scrape-recipe-description') }}</p>
+          <p>{{ $t('recipe.scrape-recipe-have-a-lot-of-recipes') }} <a :href="bulkImporterTarget">{{ $t('recipe.scrape-recipe-suggest-bulk-importer') }}</a>.</p>
```

Comment:
If you want to use router-link. This is working for me:

```suggestion
          <p>{{ $t('recipe.scrape-recipe-have-a-lot-of-recipes') }} <router-link :to="bulkImporterTarget">{{ $t('recipe.scrape-recipe-suggest-bulk-importer') }}</router-link>.</p>


```

Reply:
I see what you mean with won't update the menu, forget this